### ISSUE TITLE
Finetuning CLIP VIT32 on COCO captions

### ISSUE THREAD

**Issue by MLRadfys** [mlfoundations/open_clip#811]
Hi and thank you so much for this awesome repository!

Iam trying to finetune the openAI CLIP V32 model on the COCO dataset and observe some major overfitting, though Iam not really sure what the problem is.
I basically took the standard finetuning parameters (batch size = 256, lr = 5e-6, warmpup = 10000). 
Unfortunatley the loss functions looks like this: 

![image](https://github.com/mlfoundations/open_clip/assets/17001704/b1d19829-df85-4c56-8b29-ece9f349a761)

Another strange thing is the scale parameter. It decreases in the beginning and goes up to 100 again during the end of the training:

![image](https://github.com/mlfoundations/open_clip/assets/17001704/f8be445d-834c-4b16-89c2-31c6a391f118)

But I also have to mention that some validation metrices increases:
Validation prior-training:

image_to_text_mean_rank: 40.5519	image_to_text_median_rank: 6.0000	image_to_text_R@1: 0.2622	image_to_text_R@5: 0.4770	image_to_text_R@10: 0.5818	text_to_image_mean_rank: 56.3405	text_to_image_median_rank: 8.0000	text_to_image_R@1: 0.2268	text_to_image_R@5: 0.4399	text_to_image_R@10: 0.5386	val_loss: 1.3907	epoch: 0.0000	num_samples: 11829.0000

Post-training:
image_to_text_mean_rank: 19.8642	image_to_text_median_rank: 3.0000	image_to_text_R@1: 0.3605	image_to_text_R@5: 0.6200	image_to_text_R@10: 0.7266	text_to_image_mean_rank: 28.4263	text_to_image_median_rank: 3.0000	text_to_image_R@1: 0.3553	text_to_image_R@5: 0.6277	text_to_image_R@10: 0.7265	val_loss: 1.1578	epoch: 30.0000	num_samples: 11829.0000

Any advice would be really appreciated :-)

Thanks in advance,

M

**Comment by sean-xr** [comment]
Hello, I mght be facing the same issue, did you figure it out? Thanks for the help

**Comment by MLRadfys** [comment]
Hi!

Yes, If I remember correctly I reduced both the learning rate and the batch size to 64 :-)

Cheers,

M

**Comment by sean-xr** [comment]
Hello!

Thank you so much for the prompt and nice reply, I'm trying your
configurations now : )

May I also ask if you keep all the other configurations as the default
settings? Besides, in your first plot, did you show the val loss w.r.t the
number of epochs, not the iterations?

Thank you so much for the help, it's really appreciated.

Best regards,

Rui

**Comment by tsWen0309** [comment]
Hello, I reduced the learning rate and the batch size as you said, but I still got poor performance. Can you share the code you use for fine-tuning?

**Comment by smohan-ambarella** [comment]
+1 
Would it be possible to share the fine-tuning code ?
